iron-core/src/main/java/io/axway/iron/core/spi/file/FileTransactionStore.java

```diff
@@ -168,7 +171,7 @@ private long retrieveNextTxId() {
                     .filter(matcher -> matcher.matches() && TX_EXT.equals(matcher.group(3))) //
                     .mapToLong(matcher -> Long.valueOf(matcher.group(1))) //
                     .max() //
-                    .orElse(-1L) // in case no file exists we want nextTxId=0
+                    .orElse(-1L) // in case no file exists we want nextTxId=1 because a snapshot with no tx may have already be done to store the model
```

Comment:
i don't understand why the comment changed while the code is the same ?

Reply:
Code change reverted, not comment.
Fixed.